scripts/Avalanche.postman_collection.json

```diff
@@ -0,0 +1,3539 @@
+{
```

Comment:
Since we're going to continue referencing `https://github.com/ava-labs/avalanche-postman-collection` you can delete this file.